QuizApp/Presentation/Leaderboard/RankingCell.swift

```diff
@@ -0,0 +1,121 @@
+import UIKit
+
+class RankingCell: UITableViewCell {
+
+    static let reuseIdentifier = String(describing: RankingCell.self)
+
+    private struct CustomConstants {
+        static let textNumberOfLines = 0
+    }
+
+    private var containerView: UIView!
+    private var rankLabel: UILabel!
+    private var userLabel: UILabel!
+    private var pointsLabel: UILabel!
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+        selectionStyle = .none
+        backgroundColor = .clear
+
+        createViews()
+        styleViews()
+        defineLayoutForViews()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func createContainerView() {
+        containerView = UIView()
+        addSubview(containerView)
+    }
+
+    private func createUserLabel() {
+        userLabel = UILabel()
+        containerView.addSubview(userLabel)
+    }
+
+    private func createRankLabel() {
+        rankLabel = UILabel()
+        containerView.addSubview(rankLabel)
+    }
+
+    private func createPointsLabel() {
+        pointsLabel = UILabel()
+        containerView.addSubview(pointsLabel)
+    }
+
+}
+
+extension RankingCell: ConstructViewsProtocol {
+
+    func createViews() {
+        createContainerView()
```

Comment:
Why do you have separate functions for init of components? Can't it all be in this function?